cylc/flow/network/resolvers.py

```diff
@@ -762,11 +762,11 @@ def put_messages(
                 Information about outcome.
 
         """
-        #  TODO: standardise the task_job interface to one of the other
+        #  TODO: standardise the job_id interface to one of the other
         #        systems
```

Comment:
What does this mean/is this valid anymore?

Reply:
`git blame` flags @dwsutherland as the author of that comment. Do you recall what it means David?

Reply:
I'm afraid I don't quite remember, however, it may have been pre-universal ID.. or perhaps there was an inconsistency between mutations and hence resolvers..

Can probably remove it.

Reply:
kill it.